lib/utils.js

```diff
@@ -84,12 +84,17 @@ const opTemplate = (strings, ...keys) => {
   return result.join("");
 };
 
+const getSchemas = schema => {
+  const schemas = (Array.isArray(schema) ? schema : [schema]).filter(
+    s => typeof s === "string" && s.length > 0
+  );
+  return schemas.length > 0 ? schemas : ["public"];
```

Comment:
--> single quotes?  (as that is part of the change in the other PR)

Reply:
Can't do now, until prettier config is changed